package.json

```diff
@@ -36,7 +36,7 @@
     "package.json": "sort-package-json"
   },
   "dependencies": {
-    "solmate": "github:transmissions11/solmate#a9e3ea2"
```

Comment:
newest?

Reply:
Yes! I needed some library that is present in newer versions